cmd/createorupdate/update_test.go

```diff
@@ -0,0 +1,182 @@
+package main
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/openshift/openshift-azure/pkg/api"
+)
+
+func TestIsScaleOp(t *testing.T) {
+	tests := []struct {
+		name      string
+		cs, oldCs *api.OpenShiftManagedCluster
+		exp       map[api.AgentPoolProfileRole]scaleOp
+	}{
+		{
+			name: "no change",
+			cs: &api.OpenShiftManagedCluster{
+				Properties: &api.Properties{
+					OrchestratorProfile: &api.OrchestratorProfile{},
+				},
```

Comment:
this doesn't actually test the counts not changing

Reply:
fixed